/{runtime}/events: Include eth_tx_hash in response

ptrus opened the pull request:
Closes: https://github.com/oasisprotocol/oasis-indexer/issues/363

mitjat reviewed (approved):
Thank you! Have you tested the JSON response for an event with a corresponding tx, and for an event with no tx? (Events with no tx include one of deposit/withdrawal, IIRC. You can find them with `make psql`, then `select ... from chain.runtime_events where "type"=...`)

I can't wait for #410 to land so we get these tests/comparisons mostly automatically :)

ptrus commented on api/spec/v1.yaml:
```diff
@@ -1912,6 +1914,12 @@ components:
           example: *tx_hash_1
+        eth_tx_hash:
+          type: string
+          nullable: true
```
You are right, confirmed that `nullable` is not needed, and can just be omitted. Updated as you suggested.

Not sure why it's like that for `tx_hash` (I copied it from there :-)). Let's update that case in a separate PR, if we decide to.

ptrus commented:
> Have you tested the JSON response for an event with a corresponding tx, and for an event with no tx?

Tested now (after addressing review comments) and it works as expected in all cases (runtime event, evm runtime event, block event). There is a minor inconsistency between `tx_hash` and `eth_tx_hash`, see the other comment.
